decisions/interventions/0002-decouple-with-events.md

```diff
@@ -40,38 +40,42 @@ The above qualities point towards a decentralised way of integration and notifyi
 
 **We will use _domain_ events to decouple** from other production services.
 
-For example, our events may be:
+Our domain event payloads will contain:
 
-- intervention completed
-- service user appointment booked
-- (many others)
+- transport-specific concerns, e.g. request and tracking IDs, publishing timestamps, etc.
+- domain context, e.g. URL on how to find out more, intervention ID, service user ID, etc.
+- core domain data, which is required to make sense of the event
 
-----
+The payloads will **not** contain further data by default. **Consumers can request further details via the business API**.
 
-### Unsure about this one, help please 🙇‍♂️
+We do not intend to build event sourcing. We will **not** build event logs by default.
 
-Some consumers may need to build a local (materialised) view of events that happened in the past, even if
-they join the landscape later.
+**Examples**
 
-For this use case, **we will also create an event log**, internal to our service.
+For example, our events may be:
 
-----
+- intervention completed
+- service user appointment booked
+- (many others)
 
 ## Consequences
 
 We will build our intervention services to emit known significant domain events by default.
 
 **Benefits**
 
-- The decoupling allows developing almost all user stories without waiting for integrations.
+- The decoupling allows us developing almost all user stories without waiting on endpoints to be finished.
 - New and old systems can join as consumers any time.
-- Consumers can test their integration with us without running our service.
```

Comment:
I've removed "Consumers can test their integration with us without running our service" because if we have skinny events, our service will be potentially needed for the "further details" call.